src/main/java/org/opentripplanner/api/common/RoutingResource.java

```diff
@@ -391,6 +391,10 @@
     @QueryParam("geoidElevation")
     private Boolean geoidElevation;
 
+    /* If trip discovery mode (look out a week) should be used */
```

Comment:
This comment should be revised. "Discovery mode" is not defined, and this parameter seems to set the number of days (not a specific one-week time period). Javadoc here should probably be very similar to RoutingRequest. serviceDayLookout.